openfe/tests/protocols/test_openmm_equil_rfe_protocols.py

```diff
@@ -458,7 +461,58 @@ def test_dry_run_ligand_tip4p(benzene_system, toluene_system,
     with tmpdir.as_cwd():
         sampler = dag_unit.run(dry=True)['debug']['sampler']
         assert isinstance(sampler, MultiStateSampler)
-        assert sampler._factory.hybrid_system
+
+        # Get the htf and check
```

Comment:
is it going to be a pain to make this a fixture and make these three tests individual?

Reply:
Would have to try, but iirc tmpdir is function scoped, so you can't use it in a module scoped fixture - at least I remember this being a thing in the past. (definitely wouldn't recommend this being function scoped, it's super slow)

Reply:
ah yeah, there's [tmp_path_factory](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-factory-fixture) which has longer scope

Reply:
Done